shared/menubar/index.desktop.js

```diff
@@ -277,133 +203,132 @@ class MenubarRender extends Component<Props, State> {
       onClick={() => openApp(tab)}
     >
-      <Icon color={globalColors.darkBlue4} hoverColor={globalColors.black_75} fontSize={22} type={iconType} />
-      {!!count && <Badge badgeNumber={count} badgeStyle={{position: 'absolute', top: -6, right: -8}} />}
-    </Box>
+      <Kb.Icon color={Styles.globalColors.darkBlue4} hoverColor={Styles.globalColors.black_75} fontSize={22} type={iconType} />
+      {!!count && <Kb.Badge badgeNumber={count} badgeStyle={{position: 'absolute', top: -6, right: -8}} />}
+    </Kb.Box>
   )
 }
 
-const borderRadius = 4
-
-const stylesContainer = {
-  ...globalStyles.flexBoxColumn,
-  flex: 1,
-  position: 'relative',
-  marginTop: 13,
-  borderTopLeftRadius: borderRadius,
-  borderTopRightRadius: borderRadius,
-}
-
-const stylesTopRow = {
-  ...globalStyles.flexBoxRow,
-  alignItems: 'center',
-  backgroundColor: globalColors.darkBlue2,
-  flex: 1,
-  minHeight: 40,
-  maxHeight: 40,
-  paddingLeft: 8,
-  paddingRight: 8,
-  borderTopLeftRadius: borderRadius,
-  borderTopRightRadius: borderRadius,
-}
-
-const stylesPrivate = {
-  container: {
-    ...stylesContainer,
-    backgroundColor: globalColors.darkBlue,
-  },
-}
-
-const stylesPublic = {
-  container: {
-    ...stylesContainer,
-    backgroundColor: globalColors.white,
-  },
-}
-
-const stylesLogo = {
-  alignSelf: 'center',
-  marginBottom: 12,
-}
+const styles = Styles.styleSheetCreate({
+  widgetContainer: Styles.platformStyles({
+    common: {
+      ...Styles.globalStyles.flexBoxColumn,
+      flex: 1,
+      position: 'relative',
+      marginTop: isDarwin ? 13 : 0,
+      borderTopLeftRadius: Styles.globalMargins.xtiny,
+      borderTopRightRadius: Styles.globalMargins.xtiny,
+      backgroundColor: Styles.globalColors.darkBlue,
+    },
+  }),
+  topRow: Styles.platformStyles({
+    common: {
+      ...Styles.globalStyles.flexBoxRow,
+      alignItems: 'center',
+      backgroundColor: Styles.globalColors.darkBlue2,
+      flex: 1,
+      minHeight: 40,
+      maxHeight: 40,
+      paddingLeft: 8,
+      paddingRight: 8,
+      borderTopLeftRadius: Styles.globalMargins.xtiny,
+      borderTopRightRadius: Styles.globalMargins.xtiny,
+    },
+  }),
+  logo: Styles.platformStyles({
+    common: {
+      alignSelf: 'center',
+      marginBottom: 12,
+    },
+  }),
+  headerBadgesContainer: Styles.platformStyles({
+    common: {
+      ...Styles.globalStyles.flexBoxRow,
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+      marginLeft: 24 + 8,
+    },
+  }),
+  arrowTick: Styles.platformStyles({
+    common: {
+      height: 0,
+      width: 0,
+      position: 'absolute',
+      left: 0,
+      right: 0,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+      top: -6,
+    },
+    isElectron: {
+      borderLeft: '6px solid transparent',
+      borderRight: '6px solid transparent',
+      borderBottom: `6px solid ${Styles.globalColors.darkBlue2}`,
+    },
```

Comment:
Hmm does flow complain if these weren't in isElectron?

Reply:
Yes

Reply:
If you use the long form it's compatible

Reply:
Long form?

Reply:
This is the short hand form. If you do borderLeftWidth: 6   BorderStyle: 'solid' borderLeftColor etc they're the same. Rn doesn't support the short hand where you can define multiple props at the same time

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties